backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/types/Card.kt

```diff
@@ -0,0 +1,28 @@
+package app.ehrenamtskarte.backend.verification.webservice.schema.types
+
+data class Card constructor(
+    val totpSecret: ByteArray,
```

Comment:
Maybe List<Byte> to avoid the ugly equals and hashCode stuff

Reply:
👍🏽 Done